changes/issue-6628-macos-vuln

```diff
@@ -0,0 +1,5 @@
+* Improved vulnerability detection on MacOS to decrease false positives and added the ability to define custom rules for matching software to CPEs for vulnerability processing.
```

Comment:
> ability to define custom rules

@michalnicp can you please help me understand what we added in 4.20.0? What are the custom rules?